FordLabs/retroquest
# Refactor Team Page to use an internal Array instead of two Maps

## Overview
The current develop branch uses two maps in order to store and index the thoughts. I changed this to only use a single array, which is technically slower, but good for our purposes and actually makes the code a lot cleaner.

## Testing Instructions
All of the tests were run, including the E2E tests to make sure no functionality was lost, but feel free to test out CRUD operations and sorting of the column.
